flaky VolumetricMaxPooling test

A few times, VolumetricMaxPooling (in legacy.nn) kept failing on the continuous builds, and needed to investigate that.

So it turns out, there were these one set of indices that were in the same Pooling window that differed by less than epsilon. So, the numeric gradient was hitting boundary conditions (max-pooling is discontinuous of course).

![screen shot 2017-03-06 at 11 36 18 pm](https://cloud.githubusercontent.com/assets/1310570/23642248/2a678388-02c7-11e7-91a2-88a04aff2f34.png)
![screen shot 2017-03-06 at 11 43 53 pm](https://cloud.githubusercontent.com/assets/1310570/23642250/2c3f4272-02c7-11e7-90c7-2913ec70107f.png)

The obvious fix is to modify the test to not have the input tensor have these boundary conditions.

However, as an aside, I am not sure why the boundary condition is only hit sometimes and not always, because the same random numbers are generated always, and division is unstable but it is deterministic (right? or have i always got this wrong)

Repeat running test_legacy_nn.py will produce error occasional , more detail: 

```
======================================================================
FAIL: test_VolumetricMaxPooling (__main__.TestNN)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_legacy_nn.py", line 579, in <lambda>
    setattr(TestNN, test_name, lambda self, test=test: test(self))
  File "/Users/teaonly/Workspace/github/pytorch/test/common_nn.py", line 542, in __call__
    self._do_test(test_case, module, input)
  File "test_legacy_nn.py", line 23, in _do_test
    test_case.check_jacobian(module, input, self.jacobian_input)
  File "/Users/teaonly/Workspace/github/pytorch/test/common_nn.py", line 430, in check_jacobian
    PRECISION
AssertionError: 0.22816601327646424 not less than or equal to 1e-05

----------------------------------------------------------------------
```

--- a/test/test_legacy_nn.py
+++ b/test/test_legacy_nn.py
@@ -483,7 +483,7 @@ tests = [
                   input_size=(1, 2, 4, 4, 4)),
     OldModuleTest(nn.VolumetricMaxPooling,
                   (2, 2, 2),
-                  input_size=(2, 3, 5, 5, 5)),
+                  input=(torch.randn(2, 3, 5, 5, 5) * 1000)),
     OldModuleTest(nn.VolumetricMaxPooling,
                   (2, 2, 2, 2, 2, 2),
                   input_size=(2, 3, 5, 5, 5),

